src/parser.pegjs

```diff
@@ -48,3 +52,64 @@ digit
 
 hex_digit
 	= [0-9a-f]i
+
+// RSQL "quoted" values extesion
+// inspired by https://github.com/pegjs/pegjs/blob/master/examples/javascript.pegjs#L271
+
+string "string"
+  = '"' chars:double_string_character* '"' {
+      return chars.join("");
+    }
+  / "'" chars:single_string_character* "'" {
+      return chars.join("");
+    }
+
+double_string_character
+  = !('"' / "\\") source_character { return text(); }
+  / "\\" sequence:escape_sequence { return sequence; }
+
+single_string_character
+  = !("'" / "\\") source_character { return text(); }
+  / "\\" sequence:escape_sequence { return sequence; }
+
+escape_sequence
+  = CharacterEscapeSequence
+  / hex_escape_sequence
+  / unicode_escape_sequence
+
+CharacterEscapeSequence
```

Comment:
I would like to have it snake case called as the other rule names instead of camel case, i.e. `character_escape_sequence`.